Tribler/Core/Modules/restapi/debug_endpoint.py

```diff
@@ -342,3 +344,93 @@ def render_GET(self, request):
         request.setHeader(b'content-type', 'application/json')
         request.setHeader(b'Content-Disposition', 'attachment; filename=tribler_memory_dump_%s.json' % date_str)
         return open(dump_file_path).read()
+
+
+class DebugLogEndpoint(resource.Resource):
+    """
+    This class handles the request for displaying the logs.
+    """
+
+    def __init__(self, session):
+        resource.Resource.__init__(self)
+        self.session = session
+
+    def render_GET(self, request):
+        """
+        .. http:get:: /debug/log
+
+        A GET request to this endpoint returns the location of log file
+
+            **Example request**:
+
+            .. sourcecode:: none
+
+                curl -X GET http://localhost:8085/debug/log
+
+            **Example response**:
+
+            The content of the log file, for eg.
+            INFO    1506675301.76   sqlitecachedb:181   Reading database version...
```

Comment:
This is not what is returns, according to the code, it should return something like:

```
{'content': '...', 'max_lines': 3}
```